### PR TITLE
Project charter proofread fixes

### PR DESCRIPTION
**Pull request by dlarocque**

### CODE REVIEW

File: project-charter.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ Provide information on how the development and distribution of the Project Chart
...
 
 The intended audience of the U↑ project charter is the project sponsor and senior leadership.
 
 ## 2 PROJECT AND PRODUCT OVERVIEW
 
-U↑ is an internet-based phone system for company employees. Employees can make and receive calls over the internet using their desktop computers, smartphones, tablets, or other devices connected to the U↑ system. Each user's devices connected to U↑ will be associated with a unique business phone number for incoming and outgoing calls. Any connected device can make or receive calls using U↑ whenever and wherever necessary.
+U↑ is an internet-based phone system for company employees. Employees can make and receive calls over the Internet using their desktop computers, smartphones, tablets, or other devices connected to the U↑ system. Each user's device connected to U↑ will be associated with a unique business phone number for incoming and outgoing calls. Any connected device can make or receive calls using U↑ whenever and wherever necessary.
```
Comment (mbattad) [inline project-charter.md]: not 100% sure this needs to be fixed, I was under the impression a single user can connect multiple devices (e.g. tablet, phone, laptop, etc.) and as long as they belong to the same person they'll have the same phone number. curious how others interpreted it

**Review by mbattad — COMMENTED** [review]
looks great other than the one nitpick! may not even need to be changed, we'll see what the others think

File: project-charter.md (hunk 1 above)
Comment (BarbzCodez) [reply project-charter.md]: I interpreted it the same way as Mia. I think having multiple devices the user can use makes more sense than just having one device where they can call. Now getting a call should be blasted to all logged-in sessions (?) or log off the other session once you connect to the new one.

Comment (dlarocque) [reply project-charter.md]: Ah my bad, I just changed it without giving it much thought after Grammarly flagged the wording.

Would this be better?
```
Each of the user's devices connected to...
```

**Pull request merged**